### ISSUE TITLE
Song page with edited difficulties shows unexpected default chart

### ISSUE THREAD

**Issue by Malikil** [Reyder95/sdvx-tracker-frontend#108]
When going to a song page where the lowest diff wasn't added with the rest of them, it will default to showing the lowest one which was added **initially**.

I noticed this on KIMIDORI Streak!! (id 11) where I missed NOV on initial entry, and tested on
月見夜ラビット (id 48) where I only entered EXH initially to confirm.  
KIMIDORI is showing ADV by default, skipping over NOV, and 月見夜ラビット shows EXH by default, skipping over both NOV and ADV.

**Comment by Reyder95** [comment]
Yeah this is known. I simply just checked the first valid song entry and set that to the default. I could have checked if novice existed, then advanced existed etc but I feel that many loops would be insane. Not sure if there is a better alternative? Might be S T O O P I T
